### PR TITLE
Bump submariner-operator version

### PR DESCRIPTION
**Pull request by tpantelis**
...to pull in https://github.com/submariner-io/submariner-operator/pull/2491

Depends on https://github.com/submariner-io/submariner-operator/pull/2491

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -19,7 +19,7 @@ require (
 	github.com/submariner-io/lighthouse v0.15.0-m2.0.20230130153953-7107bd35c649
 	github.com/submariner-io/shipyard v0.15.0-m2
 	github.com/submariner-io/submariner v0.15.0-m2
-	github.com/submariner-io/submariner-operator v0.15.0-m2
+	github.com/submariner-io/submariner-operator v0.15.0-m2.0.20230206134304-33b31d9d2503
```
Comment (skitt) [inline go.mod]: This will need to be updated once the operator patch is merged.

**Review by skitt — CHANGES_REQUESTED** [review]
I’m holding this to make sure it’s updated (otherwise it could be accidentally merged before it’s ready).

File: go.mod (hunk 1 above)
Comment (tpantelis) [reply go.mod]: Yes - I was planning to do that.

Comment (skitt) [reply go.mod]: I thought you did — I just wanted to make sure the PR didn’t get two approvals and merged before it was actually ready.

Comment (tpantelis) [reply go.mod]: It's updated...

**Review by skitt — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Pull request merged**